launch.go

```diff
@@ -213,20 +232,43 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 	}
 
 	metaByte := []byte("")
-	parentID := 0
 	metaFile := "meta.json" // Write to "meta.json" file
-	metaSource := ""
+	metaLog := ""
+
+	parentBuildIDs := convertToArray(build.ParentBuildID)
 
 	// If no parent build ID, and no parent event, skip fetch meta
-	if build.ParentBuildID == 0 && event.ParentEventID == 0 {
+	if len(parentBuildIDs) == 0 && event.ParentEventID == 0 {
 		log.Printf("This build has no Parent Build and no Parent Event, so fetching Meta is skipped")
 	} else {
-		// If has parent build, fetch from parent
-		if build.ParentBuildID != 0 {
-			log.Printf("Fetching Parent Build %d", build.ParentBuildID)
-			parentBuild, err := api.BuildFromID(build.ParentBuildID)
+		if len(parentBuildIDs) > 1 { // If has multiple parent build IDs, merge their metadata
```

Comment:
We could also make this an `else if`

Reply:
We would have to duplicate the `create meta space section` or make a function out of it